src/generate_tangent.jl

```diff
@@ -0,0 +1,36 @@
+"""
+    rand_tangent([rng::AbstractRNG,] x)
+
+Returns a randomly generated tangent vector appropriate for the primal value `x`.
+"""
+rand_tangent(x) = rand_tangent(Random.GLOBAL_RNG, x)
```

Comment:
Base internally uses:
```suggestion
rand_tangent(x) = rand_tangent(Random.default_rng(), x)
```